src/screens/home/components/members-list.tsx

```diff
@@ -45,7 +49,9 @@ export function MembersList(props: MembersListProps) {
           { isExpanded && <>Show less &uarr;</> }
         </Button>
         <Link href='/join'>
-          <a><Text as='span' color='gray.400' ml='20px'>Become a member</Text></a>
+          <a><Text as='span' color='gray.400' ml='20px'>
+            { isDesktopOrLaptop ? 'Become a member' :  'Join now' }
+            </Text></a>
```

Comment:
No need for react-responsive, we can just do
```suggestion
          <a>
            <Text d={['none', 'none', 'inline', 'inline']} as='span' color='gray.400' ml='20px'>Become a member</Text>
            <Text d={['inline', 'inline', 'none', 'none']} as='span' color='gray.400' ml='20px'>Join now</Text>
          </a>
```

Reply:
Changes done as per your instructions.